tests/unit/Tumblr/StreamBuilder/StreamCursors/StreamCursorSerializerTest.php

```diff
@@ -95,6 +95,7 @@ public function testDefaultedCursor(string $cursor, ?int $user_id, ?string $exce
     {
         if ($exception) {
             $this->expectExceptionLog();
+            $this->assertNotNull($exception);
```

Comment:
I believe this is a wrong way to fix this test because `$exception` here is from a fixture. We could just as well remove the test case where it is set because it doesn't test anything.

@lucila @marco-altran

Reply:
Fixing it in #10.